x/auth/keeper/grpc_query.go

```diff
@@ -154,22 +154,8 @@ func (s queryServer) ModuleAccountByName(c context.Context, req *types.QueryModu
 	return &types.QueryModuleAccountByNameResponse{Account: any}, nil
 }
 
-// Bech32Prefix returns the keeper internally stored bech32 prefix.
-func (s queryServer) Bech32Prefix(ctx context.Context, req *types.Bech32PrefixRequest) (*types.Bech32PrefixResponse, error) {
-	bech32Prefix, err := s.k.getBech32Prefix()
-	if err != nil {
-		return nil, err
-	}
-
-	if bech32Prefix == "" {
-		return &types.Bech32PrefixResponse{Bech32Prefix: "bech32 is not used on this chain"}, nil
-	}
-
-	return &types.Bech32PrefixResponse{Bech32Prefix: bech32Prefix}, nil
-}
-
 // AddressBytesToString converts an address from bytes to string, using the
-// keeper's bech32 prefix.
+// keeper's hex prefix.
```

Comment:
Without any prefix I guess 😅

Reply:
eheh right, pushed the correction, thanks